filter_plugins/couchdb_password_hash.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import binascii
+import hashlib
+import six
```

Comment:
this introduces additional dependencies on the host running the role?

Reply:
`six` is already a dependency of the `ansible` package (via `cryptography` and `paramiko`)